worker/pkg/workflows/datasync/activities/gen-benthos-configs/benthos-builder.go

```diff
@@ -1635,8 +1689,15 @@ benthos.v0_msg_set_structured(output);
 	return jsCode
 }
 
-func constructBenthosOutput(col string) string {
-	return fmt.Sprintf(`output["%[1]s"] = fn_%[1]s(input["%[1]s"], input);`, col)
+func constructBenthosOutput(col string, source mgmtv1alpha1.TransformerSource) string {
```

Comment:
Can we update this function name to be more specific to javascript? The name is very generic, but it looks like it's used to build the javascript output object, not the benthos output.